### PR TITLE
Don't fail bloop if Java 11 installed in Mac OS

### PR DESCRIPTION
**Pull request by jvican**
This fix uses the godsend `/usr/libexec/java_home` to detect the java
home for 1.8 and use that one to run `coursier launch` (failing with a
helpful error message if it doesn't find it). The main idea here comes
from @olafurpg and some of the discussion of the bug

In reality, this means that the server must be run with Java 1.8 or it
will fail. This is a good compromise for now until we can ensure that
we are as efficient and correct in other Java versions as we're in Java
1.8. I've tested this in my machine where Java 8 is installed via jenv
(which doesn't set the java home variable directly) and Java 11 was
installed via homebrew casks. I believe this scenario stresses this fix
enough to make it valid.

For now, we only use this approach in Mac OS because libexec/java_home
only exist in OSX.

Fixes https://github.com/scalacenter/bloop/issues/743

### CODE REVIEW

File: bin/install.py [hunk 1]
```diff
@@ -112,6 +115,20 @@
...
 BUFFER_SIZE = 4096
 
+MACOS_LAUNCH_SCRIPT_CONTENTS = """
+#!/usr/bin/env sh
+
+BASE_BIN_DIR=$(dirname "$0")
+COURSIER_BIN="$BASE_BIN_DIR/%s"
+/usr/libexec/java_home -v 1.8 -F -R --exec java \
+     -Divy.home=%s -jar "$COURSIER_BIN" launch %s \
+     -r bintray:scalameta/maven \
```
Comment (olafurpg) [inline bin/install.py]: Why is this line needed?

Comment (jvican) [reply bin/install.py]: Because of lsp4s

Comment (olafurpg) [reply bin/install.py]: Isn't it released to maven central?
